src/AbstractLiveSets/LatticeLiveSets.jl

```diff
@@ -0,0 +1,51 @@
+# Lattice walkers
```

Comment:
just a placeholder reminder that we're going to need to add docstrings in this file

Reply:
Yup!